tests/integration.rs

```diff
@@ -352,10 +352,9 @@ mod subject_alt_names {
         assert!(names.collect::<Vec<_>>().is_empty());
     }
 
-    // This test reproduces (part of)
-    // https://github.com/rustls/webpki/issues/167 --- an end-entity cert where
-    // the common name is a `PrintableString` rather than a `UTF8String` cannot
-    // iterate over its subject alternative names.
+    // This test reproduces https://github.com/rustls/webpki/issues/167 --- an
```

Comment:
These changes should probably get squashed into the commit introducing the new test.